sdk/maps/maps-route-rest/samples/v1-beta/javascript/README.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
Heads up that this can be autogenerated by `dev-tool samples publish`

Reply:
Everything under samples/* can be generated from samples-dev

Reply:
Thanks for the reminder! I already doing this. But I forget to generate the latest version 😅